config/tutorial.winston.js

```diff
@@ -0,0 +1,16 @@
+const { createLogger, transports, format } = require("winston");
+
+const tutorialLogger = createLogger({
+  transports: [
+    new transports.Console({
+      level: "info",
+      format: format.combine(format.timestamp(), format.json()),
+    }),
+    new transports.Console({
+      level: "error",
+      format: format.combine(format.timestamp(), format.json()),
+    }),
+  ],
+});
+
+module.exports = { tutorialLogger };
```

Comment:
We should export logger as default. Also name of the variable should be `logger`.